jaas/src/main/java/com/griddynamics/jagger/jaas/storage/model/JobEntity.java

```diff
@@ -29,8 +31,8 @@
     private Long jobStartTimeoutInSeconds;
 
     @JsonIgnore
-    @OneToOne(mappedBy = "job", cascade = CascadeType.REMOVE)
-    private JobExecutionEntity jobExecutionEntity;
+    @OneToMany(mappedBy = "job", fetch = FetchType.EAGER, cascade = CascadeType.ALL, orphanRemoval = true)
+    private List<JobExecutionEntity> jobExecutionEntities;
```

Comment:
I've tried to remove it and got error on deleting JobEntity.
Since, there's a relation between JobEntity and JobExecutionEntity and second cannot exist without first, when you delete JobEntity you need to delete all JobExecutionEntities connected to it firstly. It seems to be wrong and ugly to me, so I think it's better to leave it as is.